types/index.d.ts

```diff
@@ -23,6 +23,15 @@ declare module 'highlight.js' {
         install: (vue: any) => void
     }
 
+    // perhaps make this an interface?
+    type RegexOptions = {
+        capture?: boolean
+    }
+
+    type RegexEitherArgs =
+        (RegExp | string)[] | [RegexOptions] |
+        [...(RegExp | string)[], RegexOptions]
```

Comment:
You can't have just options... so minimally this should be:

```
  (RegExp | string)[] | 
  [...(RegExp | string)[], RegexEitherOptions]
```

Reply:
And we can probably just inline this type, I don't think we need a custom type for a single function.